dockers/base-conda/Dockerfile

```diff
@@ -147,12 +147,33 @@ RUN \
     pip install --no-cache-dir --global-option="--cuda_ext" https://github.com/NVIDIA/apex/archive/refs/heads/master.zip && \
     python -c "from apex import amp"
 
+RUN \
+    # install FairScale
+    pip install fairscale==0.4.5 && \
```

Comment:
In the gpu job you set it to `>=`. Shouldn't they match?

Reply:
we can, but this shall be more stable, related to freezing env